module/x/gravity/abci.go

```diff
@@ -143,7 +143,7 @@ func ValsetSlashing(ctx sdk.Context, k keeper.Keeper, params types.Params) {
 	// unslashedValsets are sorted by nonce in ASC order
 	// Question: do we need to sort each time? See if this can be epoched
 	for _, vs := range unslashedValsets {
-		confirms := k.GetValsetConfirms(ctx, vs.Nonce)
+		confirms := k.GetUpdateSignerSetTxSignatures(ctx, vs.Nonce)
```

Comment:
This is kind of a mouthful. I wonder if we could call it `SignerSetTx` instead of `UpdateSignerSetTx`. That way this function would be `GetSignerSetTxSignatures`